test/e2e/util/util.go

```diff
@@ -371,95 +370,46 @@ func pathTypePrefix() *networkingv1.PathType {
 	return &p
 }
 
-func NewGateway(gatewayName, ns, secretName string, annotations map[string]string, dnsNames ...string) (*gwapiv1alpha2.Gateway, *gwapiv1alpha2.HTTPRoute) {
-	var hostnames []gwapiv1alpha2.Hostname
-	for _, dnsName := range dnsNames {
-		hostnames = append(hostnames, gwapiv1alpha2.Hostname(dnsName))
-	}
+// NewGateway creates a new test Gateway. There is no Gateway controller
```

Comment:
GitHub displays this diff in a confusing way- this change just removes the creation on an HTTPRoute from this funcion and adds a comment